Documentation/content/docs/develop_widget.md

```diff
@@ -145,6 +145,11 @@ This mixin adds an `origin` property.
 
 These mixins adds a single scale factor and a 3-component scale factor, respectively.
 
+If `scaleByDisplay` is set to true for a representation, then scale1 will be interpreted as
+a fraction of the display height. (Only for representations that support this; grep for
+"scaleByDisplay" to see which representations do.) If `scaleByDisplay` and `scaleInPixels`
+are true, then scale1 will be treated as a pixel height value.
```

Comment:
If I am following correctly, the scale is

1. In world coordinates (default)
2. In display coordinates (scaleByDisplay)
3. In pixel coordinates (scaleInPixels)

?

Would a set of constants / vtk.js enumerations work?

Reply:
The API is something worth discussing, and I like your bullet points since they provide clarity, so I'll add it in the docs.

The scale is as such:
1. Default: scale only describes world coordinate sizing
2. scaleByDisplay: the name is confusing. The scale is interpreted to be a fraction of the display height, so if the canvas is 200px and the scale is 0.05, then a sphere handle will take up 10px in height.
3. scaleInPixels: The scale is interpreted as the height in pixels.

Note this all depends on the height rather than the width, since the camera view angle (which is used for the scale calculations) determines height of the viewport.

I like the constants idea. I avoided it initially since it is an API breaking change, but after re-thinking it I'm going to go ahead with that.

Reply:
Cool, thanks @floryst

Reply:
In addition, I'm thinking of removing the 2nd functionality. I would suppose that most people will just beeline to setting pixel height/size, since that's what everyone does for sizing controls anyways. It arguably is odd if widget controls grew in size if the viewport were to be resized along the height dimension.